swift-ticketing/src/swift_ticketing/app.clj

```diff
@@ -26,7 +26,7 @@
       (GET "/ticket" [ticket_type_id]
         (handlers/get-tickets-handler db-spec ticket_type_id))
       (GET "/booking/:booking-id/status" {:keys [cookies route-params]}
-        (handlers/get-booking-status-handler db-spec (get-uid cookies) (:booking-id route-params)))
+        (handlers/get-booking-status-handler db-spec (:booking-id route-params)))
```

Comment:
I haven't written clojure web code before. @neenaoffline what is the convention for place of destructuring of route-params? I think it could be in the handler function args.
That'll simplify the routes page a lot.

Reply:
Agreed.

@zzat I think it's worth internalizing

>  It is better to have 100 functions operate on one data structure than 10 functions on 10 data structures. 

from https://web.archive.org/web/19990117034445/http://www-pu.informatik.uni-tuebingen.de/users/klaeren/epigrams.html